DittoHealth/src/test/java/live/ditto/health/ExampleUnitTest.kt

```diff
@@ -0,0 +1,17 @@
+package live.ditto.health
+
+import org.junit.Test
+
+import org.junit.Assert.*
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
+    @Test
+    fun addition_isCorrect() {
+        assertEquals(4, 2 + 2)
+    }
+}
```

Comment:
I think we can remove the sample test

Reply:
no op, for now.
until we figure out the path for other modules too.
if we decide to remove tests, then we should remove in all modules